### DISCUSSION TITLE
Enable/disable lottie splashscreen

### DISCUSSION THREAD

**Discussion by kierstejn** [timbru31/cordova-plugin-lottie-splashscreen#782] [General]
We have a system where we have several capacitor setups, where only some will use the Lottie splashscreen. 
My question is: Is there some way of disabling the splash screen without uninstalling this package? I have tried to programmatically disable the Lottie splash screen, but there is always a small white square in the center of the normal splash screen when It runs without a Lottie splash screen

**Comment by timbru31** [comment]
Right now there is no such flag. In could introduce something like a "LottieEnabled" flag which will prevent the native code from running at all. Would that help in this case? (or is the plugin config also copied/applied to all apps?)

**Reply by kierstejn** [reply]
That would be awesome to have a flag for that in the config. The plugin config is also different for each app, depending on whether it should use the lottie splash screen or not.
